cypress/integration/group1/revokeDockstoreToken.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ *    Copyright 2022 OICR, UCSC
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+import {
+  clickFirstActionsButtonPublic,
+  clickFirstActionsButtonPrivate,
+  goToTab,
+  goToUnexpandedSidebarEntry,
+  resetDB,
+  setTokenUserViewPort,
+} from '../../support/commands';
+
+describe('Test revoke token button opens confirmation dialog successfully', () => {
```

Comment:
Nit, you're testing more than the dialog opens successfully, you're also testing that you get logged out.